README.md

```diff
@@ -98,21 +98,236 @@ When using the `--legacy-purge` option, a user account on the legacy environment
 * system group administrator for a system group that the system is a member of
 * granted permissions to the system explicitly via Users-> account-> 'Systems Administered by this User'
 
-# Usage:
+# Usages:
 
+### Registering a system to Foreman
+
+This is one of the most standard workflows with bootstrap.py. This sets up the system for content / configuration (via puppet) & provisioning.
+~~~
+# ./bootstrap.py -l admin \
+    -s foreman.example.com \
+    -o "Red Hat" \
+    -L RDU \
+    -g "RHEL7/Crash" \
+    -a ak-Reg_To_Crash
+~~~
+
+### Registering a system omitting puppet setup.
+
+There are times where you wish to not install puppet, perhaps you have a differing or existing configuration management system.
+
+~~~
+# ./bootstrap.py -l admin \
+    -s foreman.example.com \
+    -o "Red Hat" \
+    -L RDU \
+    -g "RHEL7/Crash" \
+    -a ak-Reg_To_Crash \
+    --skip-puppet
+~~~
+
+### Registering a system to Foreman, for content only.
+
+This usage leverages the `--skip-foreman` switch, which does not require username/password authentication.
+
+**NOTES**
+
+ - the `--skip-foreman` switch implies `--skip-puppet`
+ - When using `--skip-foreman`, it is expected that the organization specified  (via `--organization|-o`) is specified via **LABEL**, not **NAME**.
+
+~~~
+# ./bootstrap.py -s foreman.example.com \
+    -a ak_Reg_To_Dev_EL7 \
+    -o "Red_Hat" \
+    --skip-foreman
```

Comment:
I didn't want to retroactively remove the old examples. They are still honored, so I'd rather have them documented. (Cause users will ask).